02week/pigLatin.js

```diff
@@ -8,16 +8,30 @@ const rl = readline.createInterface({
 });
 
 
-function pigLatin(word) {
-
-  // Your code here
```

Comment:
have a more detailed code plan so we know that you actually understand what you are coding. and you aren't calling your pigLatin func anywhere.